terraform/validator-sets/dev/1e5d5a74b0fd09f601ac0fca2fe7d213704e02e51943d18cf25a546b8416e9e1.node.keys.toml

```diff
@@ -1,6 +1,6 @@
 network_signing_private_key = "f9e1b4bd35cff88f047043aebedfbd310bb6ca762040ac51c28203fb1873e539"
 network_signing_public_key = "66535c0f4f9242ea8f9ff7f68b4f50b7c9d8e7961a41cf216124b33a325505ff"
-network_identity_private_key = "200000000000000078fea3aa16bf4a9c6471aeaa6fc3e0427d311503a31fe37f0245e30addf14e53"
-network_identity_public_key = "2000000000000000882a6058c64664e59415d1692b8ecbe976665b4549631c7ff15e6fbce3e2e50e"
+network_identity_private_key = "78fea3aa16bf4a9c6471aeaa6fc3e0427d311503a31fe37f0245e30addf14e53"
```

Comment:
The prefix was due to previously used bincode serialization which appends the length. We are not using bincode any more, but custom serializers, that don't require this prefix.

Reply:
@kchalkias @mgorven Many thanks for information. I just started learning libra about a month, not see full picture yet but very happy to help you guy. :)